action.yml

```diff
@@ -43,14 +43,13 @@ branding:
 runs:
   using: 'composite'
   steps:
+    - uses: actions/setup-python@v5.1.1
```

Comment:
Don't forget the caching mechanism - https://github.com/actions/setup-python?tab=readme-ov-file#caching-packages-dependencies

Reply:
still no `pip`?

Reply:
I did several experiments with that.
- When the setup-python cache option is used, it uses the host root path and even looks for the full path inside this root dir.
- I have returned to the original independent solution but have not started with the 'python' command.